.gitignore

```diff
@@ -2,9 +2,9 @@
 Thumbs.db
 
 node_modules/
-dist/
```

Comment:
hi @julianjelfs. This should remain in the `.gitignore` file, since I don't want it pushed up to git. If you remove this, I'll try and merge and publish this tonight.